airbyte-cdk/python/airbyte_cdk/sources/streams/http/auth/oauth.py

```diff
@@ -27,13 +27,15 @@ def __init__(
         refresh_token: str,
         scopes: List[str] = None,
         refresh_access_token_headers: Mapping[str, Any] = None,
+        refresh_request_body: Mapping[str, Any] = None,
```

Comment:
`/auth/oauth.py` is marked as deprecated so I wasn't sure if I needed to update this flow. Other recent PRs update both files so I did the same.